cocos/spine/assembler/simple.ts

```diff
@@ -279,11 +279,12 @@ function cacheTraverse (comp: Skeleton): void {
     vUint8Buf.set(model.vData as TypedArray);
 
     const nodeColor = comp.color;
-    if (Color.toUint32(nodeColor) !== 0xffffffff ||  _premultipliedAlpha) {
+    const opacity = comp.node._uiProps.opacity;
+    if (Math.abs(opacity - 1) > EPSILON || Color.toUint32(nodeColor) !== 0xffffffff ||  _premultipliedAlpha) {
```

Comment:
Why need to check `Math.abs(opacity - 1) > EPSILON`?

Reply:
Only when opacity equals 1, there is no need to recalculate alpha.